File: AtCorder2/ABC445B.java
import java.util.*;
import java.util.stream.*;

//ABC445B
public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    int n = scanner.nextInt();
    String[] sArray = new String[n];
    int maxSLength = 0;
    for (int i = 0; i < n; i++) {
      String s = scanner.next();
      sArray[i] = s;
      int sLength = s.length();
      if (maxSLength < sLength) {
        maxSLength = sLength;
      }
    }
    scanner.close();
    for (int i = 0; i < n; i++) {
      String currentS = sArray[i];
      int currentSLength = currentS.length();
      if (currentSLength == maxSLength) {
        System.out.println(currentS);
      } else {
        String dot =".".repeat((maxSLength - currentSLength) / 2);
        System.out.println(dot + currentS + dot);
      }
    }
  }
}

File: AtCorder2/ABC445B.py

n = int(input())
s_list = []
max_a_length = 0

for  i in range(n):
  s = input()
  s_list.append(s)
  if len(s) > max_a_length:
    max_a_length = len(s)

for current_s in s_list:
  current_s_length = len(current_s)

  if current_s_length == max_a_length:
    print(current_s)
  else:
    dot_number = (max_a_length - current_s_length) // 2
    dot = "." * dot_number
    print(dot + current_s + dot)

